App/Sagas/ImageSharingSagas.ts

```diff
@@ -20,8 +20,8 @@ import TextileNodeActions, { TextileNodeSelectors } from '../Redux/TextileNodeRe
 import { ActionType, getType } from 'typesafe-actions'
 import NavigationService from '../Services/NavigationService'
 import * as CameraRoll from '../Services/CameraRoll'
-import { IMobilePreparedFiles } from '../NativeModules/Textile/pb/textile-go'
-import { waitFor } from './WaitFor'
+import { IMobilePreparedFiles } from '@textile/react-native-protobufs'
+import { waitFor } from '@textile/react-native-wait-for'
```

Comment:
`waitFor` has nothing to do with React Native, it's just a simple JS module. Should probably name it `redux-saga-wait-for` or something.